presto-clickhouse/src/main/java/com/facebook/presto/plugin/clickhouse/optimization/function/OperatorTranslators.java

```diff
@@ -33,8 +35,8 @@ private OperatorTranslators()
     }
 
     @ScalarOperator(ADD)
-    @SqlType(StandardTypes.BIGINT)
-    public static ClickHouseExpression add(@SqlType(StandardTypes.BIGINT) ClickHouseExpression left, @SqlType(StandardTypes.BIGINT) ClickHouseExpression right)
+    @SupportedSignatures(@SqlSignature({StandardTypes.BIGINT, StandardTypes.BIGINT}))
```

Comment:
Should both of these be BIGINT ?

Reply:
Yes, one of these values represents the argument types, and one represents the return type